server/ip-tools.ts

```diff
@@ -330,9 +328,9 @@ export const IPTools = new class {
 		return (ip: string) => IPTools.checkPattern(patterns, IPTools.ipToNumber(ip));
 	}
 
-/*********************************************************
- * Datacenter parsing
- *********************************************************/
+	/*********************************************************
+ 	* Datacenter parsing
+ 	*********************************************************/
```

Comment:
```suggestion
	/*********************************************************
	 * Datacenter parsing
	 *********************************************************/
```

Reply:
It's kind of hard too see what this from the GitHub site, but I assume you moved it back by a tab or so? `indent` complained IIRC and I like the new version better, because it aligns with the functions (it's separating the class not the file).

Reply:
No, I rearranged it so the second and third-line `*`s were aligned with the first line.

`[tab][space]*` rather than `[space][tab]*` (spaces are pointless before tabs and I'm surprised eslint doesn't complain about them).

Reply:
Actually. I figured it out last night. eslint doesn't complain about `[space][tab]` because it's inside a comment, and comments aren't linted.